app/src/main/java/org/transdroid/search/gui/PrivateSitePreference.java

```diff
@@ -124,4 +160,16 @@ private void persistUserAndPass(String username, String password) {
 		setSummary(username);
 	}
 
+	private void persistCookies() {
+		final Editor edit = PreferenceManager.getDefaultSharedPreferences(getContext()).edit();
+		for (Entry<String, EditText> entry : cookies.entrySet()) {
+			final String cookieName = entry.getKey();
+			String cookieValue = entry.getValue().getText().toString();
```

Comment:
`final` too

Reply:
Can't be final because it might be changed to null bellow.